pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -792,6 +792,12 @@ class ModelResponse:
     vendor_id: str | None = None
     """Vendor ID as specified by the model provider. This can be used to track the specific request to the model."""
 
+    id: str | None = None
+    """Unique identifier for the model response, e.g. as returned by the model provider (OpenAI, etc)."""
+
+    finish_reason: str | None = None
+    """The reason the model finished generating this response, e.g. 'stop', 'length', etc."""
+
     def otel_events(self, settings: InstrumentationSettings) -> list[Event]:
```

Comment:
The issue mentions:

> These fields would be used to populate gen_ai.response.id and gen_ai.response.finish_reasons in [opentelemetry.io/docs/specs/semconv/gen-ai/gen-ai-spans#genai-attributes](https://opentelemetry.io/docs/specs/semconv/gen-ai/gen-ai-spans/#genai-attributes)

Can you please handle that here as well so this PR can close that issue?